packages/sampler/src/MoonstoneEnvironment/MoonstoneEnvironment.module.less

```diff
@@ -35,6 +35,12 @@
 		}
 	}
 
+	.storyCell > div:not([class]):not([style]):first-child {
```

Comment:
Can you add an inline comment to describe the purpose of this particular CSS incantation for future us? :)

Reply:
:mindblown:

Reply:
Done. Originally added from #1863.